docs/deploying-airbyte/deploying-airbyte.md

```diff
@@ -0,0 +1,182 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploying Airbyte
+
+The Airbyte platform is a sophisticated data integration platform that enables you to handle large amounts of data movement. If setting up an Airbyte server does not fit your usecase needs (i.e. you're using Jupyter Notebooks or iterating on an early prototype for your project) you may find the [PyAirbyte](./using-airbyte/pyairbyte/getting-started) documentation useful. To quickly deploy Airbyte on your local machine you can visit the 
+[Quickstart](./using-airbyte/getting-started/oss-quickstart) guide.
+
+:::tip
+Enterprise Customers should follow the steps outlined in our docs on [Airbyte Self-Managed Enterprise](../enterprise-setup/README.md) and the associated [implementation guide](../enterprise-setup/implementation-guide.md).
+:::
+
+## Understanding the Airbyte Deployment
+
+Airbyte is built to be deployed on top of Kubernetes in a cloud environment. We recommend deploying Airbyte using Helm and the documented Helm chart values. 
```

Comment:
Can we specify that it can run on a K8s cluster or on a single node or someone's local machine. I don't want this to be misinterpreted as this product can only be used on some mega K8s cluster or something.

Reply:
I rewrote that section, let me know if it is clearer now.